Packages/Networking/Sources/Networking/GraphQLNetworkSession.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ApolloGitHubNetworkSession.swift
+//
+//
+//  Created by Hung Dao on 22/07/2023.
+//
+
+import Foundation
+import Apollo
+import GistHubGraphQL
+import AppAccount
+import os
+
+public class GraphQLNetworkSession {
+
+    public init() {}
```

Comment:
Suggestion: Line break between variable and init, init should place below every variables